kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -757,4 +731,38 @@ private List<String> getModemsPaths() {
         return modemsPath;
     }
 
+    private Optional<String> getModemPath(Device device) throws DBusException {
+        Optional<String> mmDbusPath = Optional.empty();
+
+        if (networkManager.getDeviceType(device.getObjectPath()).equals(NMDeviceType.NM_DEVICE_TYPE_MODEM)) {
+            mmDbusPath = this.networkManager.getModemManagerDbusPath(device.getObjectPath());
+        }
+
+        return mmDbusPath;
+    }
+
+    public boolean isModemConnected(Device modemDevice) throws DBusException {
```

Comment:
Same thing here: is it talking to NM or MM? Not clear from the method signature.

Reply:
This method retrieves the NM modem path using the `getModemPath` method. Then it is used to query the status of the modem from ModemManager. So, is the connection state of a modem related to NM or MM? Are they the same?